tools/auto_analyse_raw_data.py

```diff
@@ -671,39 +663,39 @@ def generate_code(defines, code, bits_str, name="", output=sys.stdout):
                  "'data' won't work!\n")
   # Display the "normal" version's send code incase there are some
   # oddities in it.
-  for line in code["send"]:
+  for line in code["sendcomhead"] + code["send"] + code["sendcomfoot"]:
+    if line == SAFE64NOTE:
+      line = "/// @note Function should be safe up to 64 bits."
     output.write("%s\n" % line)
 
   if len(bits_str) > 64:  # Will it fit in a uint64_t?
-    code["send64+"] = [
-        "",
-        "#if SEND_%s" % def_name.upper(),
-        "/// Send a %s formatted message." % name,
-        "/// Alternative >64bit function to send %s messages" %
-        def_name.upper(),
-        "/// Status: ALPHA / Untested.",
-        "/// Where data is:",
-        "///   uint8_t data[k%sStateLength] = {0x%s};" % (
+    for line in code["sendcomhead"] + code["send64+"] + code["sendcomfoot"]:
+      if line == SAFE64NOTE:
+        line = "/// @note Function should be safe over 64 bits."
```

Comment:
I'm of two minds if we want this as a doxygen comment. Again, it's mostly feedback for the user.
What are your feelings on it?

Reply:
Agreed, not doxygen, this is for user to fix, do we want the placement of these to be changed due to not being doxygen? best at the top?

Reply:
If it's easy, that would probably be best. Otherwise I'm not fussed.